f/assignGear/f_assignGear_clothes.sqf

```diff
@@ -105,7 +105,7 @@ if(count _rig > 0) then
 	_unit addVest (selectRandom _rig);
 };
 
-if(count _glasses > 0) then
+if(count _glasses > 0 && {!((toLowerANSI goggles _unit) in (_glasses apply {toLowerANSI _x}))}) then 
```

Comment:
I wonder if `findIf` might be a little clearer what is going on, but this works so feel free to ignore this suggestion
```suggestion
if(count _glasses > 0 && (_glasses findIf {_x == goggles _unit}) == -1) then 
```
`==` is already case-insensitive so don't need `toLower(ANSI)`. And only a single iteration though the `_glasses` array instead of 2 (not that it should be big enough to matter really)

Reply:
Good call